app/helpers/application_helper/toolbar/cloud_object_store_container_center.rb

```diff
@@ -1,4 +1,27 @@
 class ApplicationHelper::Toolbar::CloudObjectStoreContainerCenter < ApplicationHelper::Toolbar::Basic
+  button_group(
+    'cloud_object_store_container_vmdb',
+    [
+      select(
+        :cloud_object_store_container_vmdb_choice,
+        'fa fa-cog fa-lg',
+        t = N_('Configuration'),
+        t,
+        :items => [
+          button(
+            :cloud_object_store_container_delete,
+            'pficon pficon-delete fa-lg',
+            N_('Remove selected Object Storage Containers'),
+            N_('Remove Object Storage Container'),
+            :url_parms => "main_div",
+            :confirm   => N_("Warning: The selected Object Storage Container and ALL related Objects will be "\
+                             "permanently removed!"),
```

Comment:
@Ladas The support check is in the [Toolbar Button code](https://github.com/Ladas/manageiq-ui-classic/blob/29e35d051703fc0b1e85ec88d14ca62d972dcad5/app/helpers/application_helper/button/generic_feature_button_with_disable.rb#L11):


In Toolbar is only specified feature.